docs/markdown/MesonCI

```diff
@@ -1,4 +1,4 @@
-# Meson CI setup
+# Meson I setup
```

Comment:
That doesn't make any sense, what are you trying to achieve?

Reply:
Why changing CI to I ?

Reply:
And also why delete the correct file extension from a markdown file?

...

This isn't October, but the first thing I thought when I saw this PR was:
https://blog.domenic.me/hacktoberfest/
https://drewdevault.com/2020/10/01/Spamtoberfest.html
https://twitter.com/shitoberfest